environment/src/test/java/com/sequenceiq/environment/service/integration/JpaTest.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.environment.service.integration;
+
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import com.sequenceiq.environment.proxy.repository.ProxyConfigRepository;
+import com.sequenceiq.environment.proxy.v1.ProxyTestSource;
+
+@DataJpaTest
+@EntityScan(basePackages = {"com.sequenceiq.flow.domain",
+        "com.sequenceiq.environment",
+        "com.sequenceiq.cloudbreak.ha.domain"})
+@Tag("outofscope")
+public class JpaTest {
```

Comment:
It is a too common name. I think we will create more test with a different entity, right?

Reply:
now it is more likely an example how to test database interactions. I am not hundred percent sure we need that, but in the future if we have more complex DB queries (@attilapalfi92  mentioned something like these recently, didn't he?)